File: Lib/Telerik/Examples/RssReader/RSS Library/RssItemImage.vb
Imports System.Text
Imports System.Xml.Serialization

Namespace CS.RSS_Library
	<XmlType("enclosure")>
	Public Class RssItemImage
		Public Sub New()

		End Sub

		<XmlAttribute("url")>
		Public Property Url() As String

	End Class
End Namespace

File: Lib/Telerik/Examples/RssReader/RSS Library/RssItemImage.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Xml.Serialization;

namespace RssReader.RSS_Library
{
    [XmlType("enclosure")]
    public class RssItemImage
    {
        public RssItemImage()
        {

        }

        [XmlAttribute("url")]
        public string Url { get; set; }

    }
}
